src/Widget.php

```diff
@@ -85,17 +85,18 @@ class="widefat"
           type="text" value="<?php esc_attr_e($scenario_id); ?>">
         </p>
         <?php
+        return '';
```

Comment:
`::form()` is expected to return a string - [there is a comment about this here](https://developer.wordpress.org/reference/classes/wp_widget/form/)